tests/unit_tests/models/test_anthropic.py

```diff
@@ -0,0 +1,22 @@
+import os
+
+import pytest
+from dotenv import load_dotenv
+
+from proxyllm.provider.anthropic.claude import AnthropicException, ClaudeAdapter
+
+load_dotenv(".env.test")
+
+anthropic_api_key = os.getenv("ANTHROPIC_API_KEY")
+
+
+def test_empty_api_credentials():
+    with pytest.raises(AnthropicException):
+        ClaudeAdapter(api_key="", auth_token="").get_completion("Sample prompt")
```

Comment:
You want to isolate this test to just an invalid api_key, since we don't know what the auth_token does

Reply:
Auth_token was removed per adrians request.